autogluon/task/tabular_prediction/predictor.py

```diff
@@ -386,6 +388,43 @@ def feature_importance(self, model=None, dataset=None, features=None, raw=True,
 
         return self._learner.get_feature_importance(model=model, X=dataset, features=features, raw=raw, subsample_size=subsample_size, silent=silent)
 
+    def refit_full(self, model=None):
+        """
+        Optimizes a model's inference time by collapsing bagged ensembles into a single model fit on all of the training data.
+        For bagged models, this process will typically result in a slight accuracy reduction and a large inference speedup.
+            The inference speedup will generally be between 10-200x faster than the original bagged ensemble model.
+            The runtime is generally 10% or less of the original fit runtime.
+        This function can also be called on non-bagged models for a slight accuracy increase and no change to inference time.
+            In this instance, the runtime will be approximately equal to the original fit runtime.
+        This process does not alter the original models, but instead adds additional models.
+        Models produced by this process will not have validation scores, as they use all of the data for training.
```

Comment:
Should we add statement addressed to users with no extra test data? (as this may be many of them)

Reply:
Added